modules/aws/agentless-gw/main.tf

```diff
@@ -27,32 +27,35 @@ resource "random_string" "gw_id" {
 }
 
 module "gw_instance" {
-  source                        = "../../../modules/aws/sonar-base-instance"
-  resource_type                 = "agentless-gw"
-  name                          = var.friendly_name
-  subnet_id                     = var.subnet_id
-  key_pair                      = var.ssh_key_pair.ssh_public_key_name
-  ec2_instance_type             = var.instance_type
-  ebs_details                   = var.ebs
-  ami                           = var.ami
-  security_groups_config        = local.security_groups_config
-  security_group_ids            = var.security_group_ids
-  role_arn                      = var.role_arn
-  additional_install_parameters = var.additional_install_parameters
-  web_console_admin_password    = var.web_console_admin_password
-  ssh_key_path                  = var.ssh_key_pair.ssh_private_key_file_path
-  binaries_location             = var.binaries_location
-  hub_sonarw_public_key         = var.hub_sonarw_public_key
-  hadr_secondary_node           = var.hadr_secondary_node
-  sonarw_public_key             = var.sonarw_public_key
-  sonarw_private_key            = var.sonarw_private_key
+  source                                 = "../../../modules/aws/sonar-base-instance"
+  resource_type                          = "agentless-gw"
+  name                                   = var.friendly_name
+  subnet_id                              = var.subnet_id
+  key_pair                               = var.ssh_key_pair.ssh_public_key_name
+  ec2_instance_type                      = var.instance_type
+  ebs_details                            = var.ebs
+  ami                                    = var.ami
+  security_groups_config                 = local.security_groups_config
+  security_group_ids                     = var.security_group_ids
+  role_arn                               = var.role_arn
+  additional_install_parameters          = var.additional_install_parameters
+  web_console_admin_password             = var.web_console_admin_password
+  web_console_admin_password_secret_name = var.web_console_admin_password_secret_name
+  ssh_key_path                           = var.ssh_key_pair.ssh_private_key_file_path
+  binaries_location                      = var.binaries_location
+  hub_sonarw_public_key                  = var.hub_sonarw_public_key
```

Comment:
It's hard for me to see if something changed here or is this just an alignment change. These changes make the code review harder and ruin the "annotate". I prefer to keep them to a minimum.